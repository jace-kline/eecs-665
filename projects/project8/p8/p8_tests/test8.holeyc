int main() {
    int a;
    int b;
    a = 6;
    a = 7;
    a = 8;
    a = a + a;
    b = a + a;
}