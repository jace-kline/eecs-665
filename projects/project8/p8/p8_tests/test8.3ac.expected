[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN main LOCALS]
a (local)
b (local)
tmp0 (tmp)
tmp1 (tmp)
[END main LOCALS]
main:       enter main
            [a] := 6
            [a] := 7
            [a] := 8
            [tmp0] := 16
            [a] := 16
            [tmp1] := 32
            [b] := 32
lbl_0:      leave main
